go-selinux/selinux.go

```diff
@@ -62,15 +62,28 @@ func ClassIndex(class string) (int, error) {
 }
 
 // SetFileLabel sets the SELinux label for this path or returns an error.
+// follow symlinks
 func SetFileLabel(fpath string, label string) error {
 	return setFileLabel(fpath, label)
 }
 
+// SetFileLabel sets the SELinux label for this path or returns an error.
+// does not follow symlinks
```

Comment:
_If the argument is a symbolic link, the label is set on the link itself. This function makes no attempt to follow the symlink._

Or somesuch.